vs/windows.undocked.props

```diff
@@ -175,8 +175,9 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <!-- Official Windows compiler and linker settings -->
   <PropertyGroup>
-    <IntDir>$(UndockedOut)obj\$(WinPlat)$(WinConfig)\$(ProjectName)\</IntDir>
-    <OutDir>$(UndockedOut)bin\$(WinPlat)$(WinConfig)\</OutDir>
+    <UndockedPlatConfig Condition="'$(UndockedPlatConfig)' == ''">$(Platform)_$(Configuration)</UndockedPlatConfig>
```

Comment:
For now, let's make this behavior change opt-in, instead of opt-out.

Reply:
that does not align with the internal onebranch change, which is opt-out, right? i think we can safely make a breaking change in this repo, and then the change (if it is indeed breaking) can be detected whenever a project updates their `undocked` submodule.